haw-pr.jenkinsfile

```diff
@@ -0,0 +1,108 @@
+            steps {
+                echo 'Starting Xvnc'
+                wrap([$class: 'Xvnc', takeScreenshot: false, useXauthority: true]) {
+                    echo 'Building and testing, using display ' + DISPLAY
+                    // dir('n4js') {
+                        script {
+                        	def envvars = [ 
+                        		'-DDISPLAY=localhost'+DISPLAY,
+                                '-DargLine="-DDISPLAY=localhost'+DISPLAY+'"'
+                        	].join(' ')
+                            def options = [
+                                '--batch-mode',
+                                //'--quiet',
+                                '--update-snapshots',
+                                '--show-version',
+                                '-Dtycho.localArtifacts=ignore',
+                                // for adjusting the script only, renable later on:
+                                 '-Dmaven.test.failure.ignore',
+                                 "--fail-at-end",
+                                '-DWORKSPACE=' + env.WORKSPACE,
+                                '-DexcludeJRE'                               
+                            ].join(' ')
+                            def profiles = [
+                                'buildProduct',
+                                'execute-plugin-tests',
+                               // 'execute-plugin-ui-tests',
+                                'execute-ecma-tests',
+                                'execute-accesscontrol-tests',
+                                'execute-smoke-tests'
+                             //'execute-hlc-integration-tests'
+                            ].join(',')
+
+                            sh """\
+                                pwd
+                                git log -n 1
+                                npm version
+                            """
+                            sh "mvn ${envvars} clean verify -P${profiles} ${options}"
+                            // works:
+                            // sh "mvn clean verify ${options}"
+                            // for quick testing: 7
+                            // sh "mvn -o clean"
+
+                          // sh "ls -Ral builds/org.eclipse.n4js.product.build/target/repository/"
+                        }
+                    // } // end dir
+                } // end wrap
+            } // end steps
+        } // end stage
+        /*
+        stage('Deploy') {
+            steps {
+                echo 'Deploying....'
+            }
+        }*/
+    }
+
+    post {
+        always {
+         //   archiveArtifacts allowEmptyArchive: true, artifacts: '**/builds/**/target/products/*.zip'
+         //   archiveArtifacts allowEmptyArchive: true, artifacts: '**/tools/**/target/n4jsc.jar'
+         //   archiveArtifacts allowEmptyArchive: true, artifacts: '**/logs/*.log'
+         //   archiveArtifacts allowEmptyArchive: true, artifacts: '**/tests/**/target/**/*-output.txt'
```

Comment:
Delete? (Also last line below)

Reply:
Also here I want to maybe archive some artifacts once I really clean the build